src/extension.ts

```diff
@@ -171,7 +171,7 @@ let startExtemporeInTerminal = () => {
     let sharedir = getExtemporePath();
 
     if (!sharedir) {
-        vscode.window.showErrorMessage("Extempore: can't find extempore folder.");
+        vscode.window.showErrorMessage("Extempore: can't find extempore folder. Set extempore.sharedir in the VSCode settings.");
```

Comment:
This is the only substantial change in this file, the rest was the work of the autoformatter. Sorry for the noise.